releases/release-1.20/exceptions.yaml

```diff
@@ -0,0 +1,24 @@
+# Exceptions requested in 1.20
+# Google Group: https://groups.google.com/forum/#!forum/kubernetes-sig-release
+# Release Team Lead: Kirsten Garrison (@kikisdeliveryservice)
```

Comment:
This should match the RT Leads and Lead Shadows.

Example: https://github.com/kubernetes/sig-release/blob/db606943bb342c1ab55cbc4b7809a6e7cb6ebd6d/releases/release-1.19/exceptions.yaml#L3-L4

Reply:
Oh gosh DUH. Was rushing will update. 🤦